Kickstarter-iOS/Views/Transitions/RewardPledgePushTransitionAnimator.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+public protocol RewardPledgeTransitionAnimatorDelegate: AnyObject {}
+
+public class RewardPledgePushTransitionAnimator: NSObject, UIViewControllerAnimatedTransitioning {
+  public func transitionDuration(
+    using _: UIViewControllerContextTransitioning?
+  ) -> TimeInterval {
+    return 0.3
+  }
+
+  public func animateTransition(using transitionContext: UIViewControllerContextTransitioning) {
+    guard let toView = transitionContext.view(forKey: .to) else { return }
```

Comment:
In the blog post examples there were delegate and vc properties on the animators but I think that that is unnecessary and will instead `guard let` them here.